codechallenge/src/main/java/com/tradeshift/homeController.java

```diff
@@ -0,0 +1,35 @@
+package com.tradeshift;
+
+import com.tradeshift.model.HelloResult;
+import com.tradeshift.model.MessageModel;
+import com.tradeshift.service.HelloService;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Controller;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+
+@Path("/")
+@Controller
+public class HomeController {
+
+    @Autowired
+    private HelloService helloService;
+
+    @GET
+    @Path("hello")
+    @Produces(MediaType.APPLICATION_JSON)
+    public HelloResult Index(@QueryParam("name") String name){
```

Comment:
java naming conventions http://www.oracle.com/technetwork/java/codeconventions-135099.html state that method names start with a lower case character.